### DISCUSSION TITLE
Week 4

### DISCUSSION THREAD

**Discussion by auroranavas** [ISPP-G5/NexONG#69] [Knowledge Database]
### FEEDBACK SOBRE LA PRESENTACIÓN
_Disclaimer: todos los integrantes del grupo 5 que atienden a clase recopilan el feedback dado por los profesores y compañeros, sin embargo, solo una persona lo sube para evitar información duplicada_
- Resumir y meter el análisis de competidores. ¿Hay nuevos? ¿Quiénes eran?
- Poner el rendimiento individual (rosco de Clockify)
- Poner gestión de usuarios pilotos. ¿Tenemos las encuestas? ¿Cuántas son? ¿Cómo vamos a tener en cuenta el feedback?
- Poner QR a la landing page al final de la presentación
- Poner planificación del Sprint 3
- MVP: ¿Qué es lo que hace que Manos Abiertas quiera usar nuestra aplicación? 
- Explicar criterios de la autodefensa -> "Creemos que hemos cubierto un xx% porque..."
- Tener en cuenta como riesgo las estimaciones de tiempo positivas
- Elevator pitch
- Orden de la presentación

**Comment by pabpercab1** [comment]
De otros grupos:
- Estimación de GitHub actions (Tiempo y licencias)
- Análisis de competidores esencial (3 primeros)
- Elevator pitch (Que queremos vender? Vendérselo a la audiencia)
- Mantenimiento: Cuidado, que es coste de operación (lo que cuesta mantenerlo, mantenimiento aumentativo, mantenimiento correctivo) CAPEX OPEX
- Orden de la presentación (Píldoras teóricas y Müller)
- Añadir triángulos hacia arriba o hacia abajo según como haya mejorado o empeorado el rendimiento, etc.
- TCO mensual con usuarios
